tests/basictests.cpp

```diff
@@ -569,32 +575,233 @@ bool skyprophet_test() {
+    return true;
+  }
+
+  bool twitter_default_profile() {
+    // Print users with a default profile.
+    set<string_view> default_users;
+    document doc = document::parse(get_corpus(JSON_TEST_PATH));
+    for (document::object tweet : doc["statuses"].as_array()) {
+      document::object user = tweet["user"];
+      if (user["default_profile"]) {
+        default_users.insert(user["screen_name"]);
+      }
+    }
+    if (default_users.size() != 86) { cerr << "Expected twitter.json[statuses][user] to contain 86 default_profile users, got " << default_users.size() << endl; return false; }
+    return true;
+  }
+
+  bool twitter_image_sizes() {
+    // Print image names and sizes
+    set<tuple<uint64_t, uint64_t>> image_sizes;
+    document doc = document::parse(get_corpus(JSON_TEST_PATH));
+    for (document::object tweet : doc["statuses"].as_array()) {
+      auto [media, not_found] = tweet["entities"]["media"];
+      if (!not_found) {
+        for (document::object image : media.as_array()) {
+          for (auto [key, size] : image["sizes"].as_object()) {
+            image_sizes.insert({ size["w"], size["h"] });
+          }
+        }
+      }
+    }
+    if (image_sizes.size() != 15) { cerr << "Expected twitter.json[statuses][entities][media][sizes] to contain 15 different sizes, got " << image_sizes.size() << endl; return false; }
+    return true;
+  }
+
+  bool run_tests() {
+    if (!object_iterator()) { return false; }
+    if (!array_iterator()) { return false; }
+    if (!object_iterator_empty()) { return false; }
+    if (!array_iterator_empty()) { return false; }
+    if (!string_value()) { return false; }
+    if (!numeric_values()) { return false; }
+    if (!boolean_values()) { return false; }
+    if (!null_value()) { return false; }
+    if (!document_object_index()) { return false; }
+    if (!object_index()) { return false; }
+    if (!twitter_count()) { return false; }
+    if (!twitter_default_profile()) { return false; }
+    if (!twitter_image_sizes()) { return false; }
+    return true;
+  }
+}
+
 int main() {
   // this is put here deliberately to check that the documentation is correct (README),
   // should this fail to compile, you should update the documentation:
   if (simdjson::active_implementation->name() == "unsupported") { 
     printf("unsupported CPU\n"); 
   }
   std::cout << "Running basic tests." << std::endl;
-  if(!json_issue467())
-    return EXIT_FAILURE;
-  if(!stream_test())
-    return EXIT_FAILURE;
-  if(!stream_utf8_test())
-    return EXIT_FAILURE;
-  if(!number_test_small_integers())
-    return EXIT_FAILURE;
-  if(!stable_test())
-    return EXIT_FAILURE;
-  if(!bad_example())
-    return EXIT_FAILURE;
-  if(!number_test_powers_of_two())
-    return EXIT_FAILURE;
-  if(!number_test_powers_of_ten())
-    return EXIT_FAILURE;
-  if (!navigate_test())
-    return EXIT_FAILURE;
-  if (!skyprophet_test())
+  // if(!json_issue467())
```

Comment:
There is a bunch of tests here that are commented out which is a bit troubling...? If the tests are no longer necessary, they should be removed... but if they are relevant, then they should run.

Reply:
Oh hell! I do that sometimes when I have a test failure, and forgot to change it back. I'll fix it.

Reply:
(To be clear, I comment out the successful tests in those cases, not the failing ones :))

Reply:
> I comment out the successful tests in those cases, not the failing ones 

Where is the fun in that?